evap/staff/templates/staff_semester_view.html

```diff
@@ -11,75 +12,122 @@
-        &nbsp;
-        <a href="{% url "evap.staff.views.semester_lottery" semester.id %}" class="btn btn-sm btn-default">{% trans "Lottery" %}</a>
-        &nbsp;
-        <a href="{% url "evap.staff.views.semester_todo" semester.id %}" class="btn btn-sm btn-default" {{ disable_if_archived }}>{% trans "Todo" %}</a>
-        &nbsp;
-
-        <div class="btn-group">
-            <a href="{% url "evap.rewards.views.semester_reward_points" semester.id %}" class="btn btn-sm btn-default">{% trans "Reward Points" %}</a>
-            <div class="btn-group btn-toggle">
-                <a href="{% url "evap.rewards.views.semester_activation" semester.id "on" %}" class="btn btn-sm btn-default {% if rewards_active %}active{% endif %}">{% trans "On" %}</a>
-                <a href="{% url "evap.rewards.views.semester_activation" semester.id "off" %}" class="btn btn-sm btn-default {% if rewards_active|is_false %}active{% endif %}">{% trans "Off" %}</a>
-            </div>
+    {% if num_courses > 0 %}
+    <div class="panel panel-info">
+        <div class="panel-heading">
+            <h3 class="panel-title">{% trans "Overview" %}</h3>
         </div>
-    </div>
-
-    {% if courses_by_state %}
-        <ul class="nav nav-tabs">
-            {% for state, courses in courses_by_state %}
-                <li><a href="#tab{{ forloop.counter }}" data-tab="{{ forloop.counter }}" data-toggle="tab">{{ state|statename }} ({{ courses|length }})</a></li>
-            {% endfor %}
-        </ul>
-
-            <div class="tab-content">
-            {% for state, courses in courses_by_state %}
-                <div id="tab{{ forloop.counter }}" class="tab-pane">
-                <table class="table table-striped">
+        <div class="panel-body">
+            <table class="table">
+                <thead>
+                    <tr>
+                        <th class="col-sm-3">{% trans "Evaluation periods" %}</th>
+                        <th class="col-sm-3">{% trans "Finished evaluations" %}</th>
+                        <th class="col-sm-3">{% trans "Reviewed comments" %}</th>
+                        <th class="col-sm-3">{% trans "Participation" %}</th>
+                    </tr>
+                </thead>
                 <tbody>
-                {% for course in courses %}
                     <tr>
-                        {% include_staff_semester_view_course semester state course disable_if_archived %}
+                        <td>{{ first_start|date:"SHORT_DATE_FORMAT" }} - {{ last_end|date:"SHORT_DATE_FORMAT" }}</td>
+                        <td>
+                            {% include_progress_bar num_courses_evaluated num_courses %}
+                        </td>
+                        <td>
+                            {% include_progress_bar num_comments_reviewed num_comments %}
+                        </td>
+                        <td>
+                            {% include_progress_bar num_votes num_enrollments_in_evaluation %}
```

Comment:
actually here the percentage is more interesting than the absolute numbers. but having the three progress bars there looks really cool...

still, could you show the percentage in plain sight?